platform/android/src/style/sources/source.cpp

```diff
@@ -93,6 +93,23 @@ namespace android {
         return attribution ? jni::Make<jni::String>(env, attribution.value()) : jni::Make<jni::String>(env,"");
     }
 
+    void Source::setPrefetchZoomDelta(jni::JNIEnv& env, jni::String& delta) {
```

Comment:
@tobrun Optional `Added in API level 24` :disappointed:. Could we use `Integer` instead of string, so that `Nullable` can be used on Java side?